### DISCUSSION TITLE
有点好奇和[halo博客]是什么关系

### DISCUSSION THREAD

**Discussion by stars-one** [ikaros-dev/ikaros#516] [Q&A] [answered]
看到文档和系统后台的截图,都和[halo-dev/halo: 强大易用的开源建站工具。](https://github.com/halo-dev/halo)有点像

是不是都是用的同个技术框架?

**Comment by chivehao** [comment, marked as answer]
感谢您的关注
`ikaros` 和 `halo` 他们官方没有关系，
只是技术上很多方案都是参考的 `halo` ，比如插件的实现方案、自定义模型的实现方案，
有很多相同的地方，也有很多不同的地方

相同的地方比如：插件的实现方案、自定义模型的实现方案、前后端混合编译的方案
不同的地方比如：`ikaros`底层数据存储是基于多表存储的，而`halo`的底层数据存储是基于一张表抽象的统一模型

`halo`是非常优秀的软件，我们尽量学习`halo`优秀的地方，向`halo`看齐（如果有成熟的方案存在、那么就没必要重复造轮子）

同时也非常欢迎感兴趣的朋友加入一起开发
